components/director/internal/tenantmapping/mapper_for_system_auth_test.go

```diff
@@ -0,0 +1,307 @@
+package tenantmapping_test
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/compass/components/director/internal/tenantmapping"
+
+	"github.com/google/uuid"
+	systemauthmock "github.com/kyma-incubator/compass/components/director/internal/domain/systemauth/automock"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	tenantmappingmock "github.com/kyma-incubator/compass/components/director/internal/tenantmapping/automock"
+	"github.com/kyma-incubator/compass/components/director/pkg/str"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestMapperForSystemAuthGetTenantAndScopes(t *testing.T) {
+	t.Run("GetTenantAndScopes returns error when unable to get SystemAuth from the service", func(t *testing.T) {
```

Comment:
minor: for me it is easier to read tests if we starts from happy paths. In addition to that `GetTenantAndScopes` appears twice in the test name.

Reply:
Done